src/Bybit/Utils.jl

```diff
@@ -0,0 +1,57 @@
+# Bybit/Utils
+
+function Serde.deser(
+    ::Type{<:AbstractAPIsData},
+    ::Type{<:DateTime},
+    x::String,
+)::DateTime
+    return unix2datetime(parse(Int64, x) * 1e-3)
+end
+
+function Serde.deser(
+    ::Type{<:AbstractAPIsData},
+    ::Type{<:Maybe{NanoDate}},
+    x::Int64,
+)::NanoDate
+    return unixnanos2nanodate(x * 1e6)
+end
+
+function Serde.deser(
+    ::Type{<:AbstractAPIsData},
+    ::Type{<:Maybe{NanoDate}},
+    x::String,
+)::NanoDate
+    return unixnanos2nanodate(parse(Int64, x) * 1e6)
+end
+
+function Serde.deser(
+    ::Type{<:BybitAPIError},
+    ::Type{<:Maybe{NanoDate}},
+    x::Int64,
+)::NanoDate
+    return unixnanos2nanodate(x * 1e6)
+end
+
+function Serde.SerQuery.ser_value(::Type{<:BybitCommonQuery}, ::Val{:timestamp}, v::Any)::Nothing
+    return nothing
+end
```

Comment:
There's a special function in Serde.jl `ser_ignore_field` that is more appropriate for this case.